### DISCUSSION TITLE
dataset problems and reproduce problems

### DISCUSSION THREAD

**Discussion by 1085052159** [STVIR/pysot#478] [Reproduce]
大佬您好，非常感谢你们开源了这个框架，在使用这个框架完成自己工作的时候，碰到了一些问题。
系统：ubuntu18.04
显卡：3090 * 2
内存：64G
CPU：共24核48线程
硬盘：4T机械
后续所有问题都是针对r50模型
问题1
我在使用你们官方模型测试的时候，在GTX1080上测试速度只有10左右，在3090上测试也只有20fps，和你们官方提供的速度相差较大
问题2
我在双卡3090上训练时，单卡batch_size设置为64，其余参数保持一致，然后有4个线程读取数据，在前几百次迭代的时候很快，每个迭代大概0.8s，当继续训练时，后续的每个迭代需要1.5s甚至更长，机器温度正常，当停止重新运行之后又会这样，您知道是什么原因导致这个问题吗
问题3:
我在双卡3090上复现你们的论文时，单卡batch_size设置为64，其余参数保持一致，我复现的最好模型在OTB2015上只有0.67左右，在VOT2018上的EAO只有0.36左右，这个是因为什么问题导致的
期待您的耐心解答，谢谢

**Comment by BOTBIT1** [comment]
我也遇到了同样的问题，请问是怎么解决的

**Comment by ZhiyuanChen** [comment]
> 我在双卡3090上训练时，单卡batch_size设置为64，其余参数保持一致，然后有4个线程读取数据，在前几百次迭代的时候很快，每个迭代大概0.8s，当继续训练时，后续的每个迭代需要1.5s甚至更长，机器温度正常，当停止重新运行之后又会这样，您知道是什么原因导致这个问题吗

这个应该是因为backbone在warmup阶段不参与训练，所以起始速度很快。backbone参与训练之后速度显著延长是正常的。

有关其他问题，我也不知道我是为什么……
